policy/test_submount.te

```diff
@@ -0,0 +1,39 @@
+#################################
+#
+# Policy for testing NFS crosmnt
+#
+
+gen_require(`
+	attribute file_type;
+	type nfs_t;
+')
+
+# A domain that can access NFS files/directories
+type test_readnfs_t;
+domain_type(test_readnfs_t)
+unconfined_runs_test(test_readnfs_t)
+domain_obj_id_change_exemption(test_readnfs_t)
+typeattribute test_readnfs_t testdomain;
+
+# Allow execution of helper programs
+corecmd_exec_bin(test_readnfs_t)
+domain_exec_all_entry_files(test_readnfs_t)
+libs_use_ld_so(test_readnfs_t)
+libs_use_shared_libs(test_readnfs_t)
+libs_exec_ld_so(test_readnfs_t)
+libs_exec_lib_files(test_readnfs_t)
+
+# Allow this domain to be entered from sysadm domain
+miscfiles_domain_entry_test_files(test_readnfs_t)
+userdom_sysadm_entry_spec_domtrans_to(test_readnfs_t)
+corecmd_bin_entry_type(test_readnfs_t)
+sysadm_bin_spec_domtrans_to(test_readnfs_t)
+
+# Allow the domain to search all dirs so that ls works
+# no matter where testsuite is run from.
+allow test_readnfs_t file_type:dir { search getattr };
+
+# Allow the domain to read NFS mounted files/directories
+allow test_readnfs_t nfs_t:fd use;
+allow test_readnfs_t nfs_t:file read_file_perms;
+allow test_readnfs_t nfs_t:dir list_dir_perms;
```

Comment:
These can likely be allowed via fs_read_nfs_files(test_readnfs_t) rather than raw allow rules.